modules/zstream-4-16-54-bug-fixes.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * release_notes/ocp-4-16-release-notes.adoc
+
+
+:_mod-docs-content-type: REFERENCE
+[id="zstream-4-16-54-bug-fixes_{context}"]
+= Bug fixes
+
+[role="_abstract"]
+The following bugs are fixed for this release:
+
+* Before this update, a Kube State Metrics (KSM) deny-list had typographical errors as demonstrated in the following example:
++
+[source,terminal,subs=""verbatim""]
+----
+--metric-denylist=
+          ^kube_secret_labels$,
+          ^kube_.+_annotations$
+          ^kube_customresource_.+_annotations_info$,
+          ^kube_customresource_.+_labels_info$,
+----
++
+With this release, a missing comma is now included as shown in the following example:
++
+[source,terminal,subs=""verbatim""]
+----
+--metric-denylist=
+          ^kube_secret_labels$,
+          ^kube_.+_annotations$,
+          ^kube_customresource_.+_annotations_info$,
+          ^kube_customresource_.+_labels_info$,
+----
++
+As a result, the entries are correctly separated. (link:https://issues.redhat.com/browse/OCPBUGS-64581[OCPBUGS-64581])
```

Comment:
I wonder do we really need to show the typo like this? Seems probably the user wouldn't be looking at it and presumably was impacted by something not working as a result of the typo, rather than caring about the typo itself.